src/scene/turret/MissileT1.gd

```diff
@@ -1,11 +1,5 @@
 extends "res://scene/turret/Turrets.gd"
 
-var missile1_node;
-var missile2_node;
-var isLeftMissileFired = false;
-
-signal fire_missile(base_position, enemy, damage);
-
```

Comment:
Do these need to be moved? Seems like they should still belong to the child class.

Reply:
Fixed, placed missile-specific process back in the missile script instead of the general script.